packagingbuild/centos8/Dockerfile

```diff
@@ -85,13 +87,13 @@ RUN yum -y install nc net-tools
 #
 
 # Install development tools and python for EL8
-RUN yum -y install python3 python2 python3-devel rpmdevtools && \
+RUN yum -y install python3 python2 python2-devel gcc python3-devel rpmdevtools && \
```

Comment:
`gcc` is already installed before, - see line `33` of the same Dockerfile. See the template file as well.

I guess it somehow appeared here by copy-paste or whatever else similar reason.